completions/sv.zsh

```diff
@@ -35,7 +35,13 @@ cmds)
         check
     ret=0;;
 args)
-    services=( /var/service/*(-/N:t) )
+    if [[ $BUFFER == sudo\ *
+        || $BUFFER == doas\ *
+        || $BUFFER == su\ *-c* ]] then
```

Comment:
```suggestion
    if [[ $BUFFER == "sudo "*
        || $BUFFER == "doas "*
        || $BUFFER == "su "*-c* ]] then
```
does this work?

Reply:
Yes, those should be equivalent, just a style difference.  Wasn't sure which one is preferred, as similar globs in the repo don't have spaces. I'll update it.